simples3.go

```diff
@@ -160,16 +163,20 @@ func (s3 *S3) getClient() *http.Client {
 }
 
 func (s3 *S3) getURL(bucket string, args ...string) (uri string) {
+	path := bucket
```

Comment:
Since this is an internal function, we can rename inside the function param itself to path.